README.md

```diff
@@ -85,7 +85,7 @@ import "@aurodesignsystem/auro-radio";
 <!-- The below code snippet is automatically added from ./../apiExamples/basic.html -->
 
 ```html
-<auro-radio id="radio1" label="Yes" name="radioDemo" value="yes"></auro-radio>
+<auro-radio label="Yes" name="radioDemo" value="yes"></auro-radio>

```

Comment:
`id` is a required part of the html5 spec for all form elements. We have to include it here and have it passed down to the inner `input` element.